src/new.rs

```diff
@@ -13,6 +12,90 @@ pub enum Template {
     App,
 }
 
+pub fn init(
```

Comment:
We could remove the duplication by having the `new` function call the `init` function

```rust
init(
  template,
  description,
  name,
  path,
  version,
)
```